src/main/java/io/kestra/plugin/notifications/sentry/SentryAlert.java

```diff
@@ -107,6 +120,12 @@ public class SentryAlert extends Task implements RunnableTask<VoidOutput> {
     @NotBlank
     protected String dsn;
 
+    @Schema(
+        title = "Sentry endpoint type"
+    )
+    @PluginProperty(dynamic = true)
+    protected EndpointType endpointType = EndpointType.ENVELOP;
```

Comment:
```suggestion
    @Builder.Default
    protected EndpointType endpointType = EndpointType.ENVELOPE;
```

Withou the Lombok annotation `@Builder.Default` the default will not be correctly used.